action/store/postgres.py

```diff
@@ -117,6 +117,12 @@ def _clean_column_default(value: str) -> Any:
             # Unwrap value cast to text as a string
             value = value[0 : -len("::text")]
             if value[0] == value[-1] == "'" and value.count("'") == 2:
-                value = value[1:-1]
-
-        return value
+                return value[1:-1]
+        elif value.isnumeric():
+            return int(value)
+        elif value.lower() == "true":
+            return True
+        elif value.lower() == "false":
+            return False
+        else:
+            return value
```

Comment:
```suggestion

        return value
```

Redundant trailing `else`